### ISSUE TITLE
not accepting calls

### ISSUE THREAD

**Issue by duportail** [HelloVolla/ubuntu-touch-beta-tests#88]
**Details about installed OS version**
(Info about which OTA/Image you are reporting this issue for, for example: Ubuntu 16.04 (2020-W44) on Release Candicate channel.)

**Describe the bug**
The phone does not accept calls anymore.
Only after reboot it accept calls.
I have now configured to never sleep. I hope the error is there to find

**Steps to reproduce the bug**
(The following is an example:)
2. Scroll down to '...'
3. Click on '...'

**Expected behavior**
(A clear and concise description of what you expected to happen.)

**Screenshots**
(If applicable, add screenshots here to better explain your problem.)

**Additional context**
(If applicable, add any other context about the problem here, such as log file attachments.)

**Comment by duportail** [comment]
setting sleep on inactivity  to never seems to solve this problem

**Comment by duportail** [comment]
problem is back. Today the phone does not accept calls or sms. Very fustrating when using in a company. People call, but get no answer.

**Comment by UniversalSuperBox** [comment]
Hi @duportail!

As mentioned in your similar report on the Ubuntu Touch bug tracker, I believe this issue is already tracked along with https://github.com/ubports/ubuntu-touch/issues/1629. At the very least, the two things have a similar root cause. Thanks for helping us track down the cause with your report, but we're still working on it. You can subscribe to notifications on that report if you would like to be notified when something changes.

Thanks again for your report.

**Comment by wurzer** [comment]
Does this occur on a stable channel with OTA 15? I just can confirm, that in some cases audio our is lost after frequent calls. Rebooting the device helps We are working on that issue.
